pkg/kube/util/names/docker_source.go

```diff
@@ -0,0 +1,27 @@
+package names
+
+import "errors"
+
+// GetDockerSourceName returns the name of the docker image
+// More info:
+//   * https://kubernetes.io/docs/concepts/containers/images
+//   * <hub-user>/<repo-name>[:<tag>]
+//   * ACCOUNT.dkr.ecr.REGION.amazonaws.com/imagename:tag
+//
+// prefix: [<host>[:<port>]/][<repo>/]
```

Comment:
Should this be 
```suggestion
// prefix: [<host>[:<port>]/][<org>/]
```
?